### ISSUE TITLE
The log versus linear debate, revisited for tellurics

### ISSUE THREAD

**Issue by gully** [gully/blase#24]
There's a tradeoff in whether to take the `log` of the model flux before processing, or simply leave them as-is, in _linear_ fluxes.  I chose the latter for stellar spectra for various reasons (that I still think are valid).

However, the calculus for tellurics is different.  Telluric spectra---as produced by `TelFit` at least---have the continuum set to exactly 1.0 everywhere, with no additional continuum opacity to deal with.  Tellurics also have have deep saturated lines.  These two differences tip the balance of the scales towards using `log` rather than linear, ideally.

The problem is that I've already coded up the stellar emulator for linear and it may be awkward to do one in linear and one in log.  

What to do! I'm not sure.

**Comment by gully** [comment]
By the way, one may be wondering "But hey, data can be negative, isn't that a problem?"  The answer is no.  We "leave that as an exercise for the reader".

**Comment by gully** [comment]
Neat experiment:

I ran the log and linear emulators for a PHOENIX spectrum to see the differences.  Takeaway: the differences between log and linear are much much lower (10x or more) than the residual between the native precomputed PHOENIX and log model.  

![image](https://user-images.githubusercontent.com/860227/159035822-7f68a49d-95eb-4a49-a8f0-5644e4bbe818.png)

So it doesn't matter much which one you use, because the residual is dominated by other differences.

We see much better performance for telluric lines though.  The main difference is how to handle saturated lines, as we predicted.
